oxd-gen-client/src/main/java/io/swagger/client/Pair.java

```diff
@@ -2,19 +2,17 @@
  * oxd-server
  * oxd-server
  *
- * OpenAPI spec version: 4.0.beta
+ * OpenAPI spec version: 4.0
  * Contact: yuriyz@gluu.org
  *
  * NOTE: This class is auto generated by the swagger code generator program.
  * https://github.com/swagger-api/swagger-codegen.git
  * Do not edit the class manually.
  */
 
-
 package io.swagger.client;
 
-@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2019-07-23T08:29:06.952Z")
-public class Pair {
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2020-02-13T06:19:07.893Z[GMT]")public class Pair {
```

Comment:
Would you please check whether there is way to skip `date` during generation? Hopefully generator has such option. These dates changes creates a lot of noise.

Reply:
This is only possible by generating  Java Client Offline with `-D hideGenerationTimestamp=true`.
We have done that by following below steps. Now there is no `date` in generated code.

1. Dowloaded codegen stable version source code from https://github.com/swagger-api/swagger-codegen/releases/tag/v3.0.16

2. Build it

3. Generate java client using below command
`java -jar swagger-codegen-cli.jar generate -i swagger.yaml -D hideGenerationTimestamp=true -l java`